packages/react/src/ExtensionDisplay/ExtensionDisplay.tsx

```diff
@@ -75,7 +75,7 @@ export function ExtensionDisplay(props: ExtensionDisplayProps): JSX.Element | nu
   return (
     <BackboneElementDisplay
       path={props.path}
-      value={{ type: typeSchema.name, value: props.value }}
+      value={{ type: typeSchema.type as string, value: props.value }}
```

Comment:
Can we make `InternalTypeSchema.type` and perhaps `InternalTypeSchema.url` required now to avoid having to cast them to string?

Reply:
Those get assigned from their SD equivalents which don't allow `undefined`
The same applies to `InternalTypeSchema.kind`

Reply:
I looked into this, and the reason they are optional is due to "inner" types.

For example, the FHIR `Patient` type ends up being multiple Medplum types:

1. `Patient`
2. `PatientCommunication`
3. `PatientContact`
4. `PatientLink`

Those inner types do not have a FHIR `type`, `url`, or `kind`.